### DISCUSSION TITLE
Split job waiting for resources to become available

### DISCUSSION THREAD

**Discussion by jakecroft** [nextpyp/.github#131] [General]
Hello, I am trying to get a preprocessing job to run and the launch job is successful but the split job perpetually gets stuck in a state of "The job is waiting for more resources to become available". I am requesting the same amount of computational resources as has worked in the past, and I have tried adjusting it to request fewer threads, but it didn't help. I also tried restarting the computer. I am running version 0.6.4 on a workstation in standalone mode (And nothing else is running on the workstation).  Any ideas of what to try would be appreciated.

Best,
Jake Croft

**Comment by cuchaz** [comment]
Hi Jake,

There's a debug tool built into nextPYP for situations like this one. If you click on your account name in the upper right corner, it should take you to a page with a link named "Administration". Click on that to go to the admin page and navigate to the "Standalone Jobs" tab. That will show you all the scheduled and running jobs at that moment, along with their resource allocations. There's also a resource allocation summary at the top. Hopefully that page can show which job is causing the bottleneck. It should look something like the attached screenshot.

![standalone-jobs](https://github.com/user-attachments/assets/a67b19ce-45db-4756-acb9-a2941a86ffbd)

**Reply by jakecroft** [reply]
Ahhh thanks, this was helpful for diagnosing the problem. NextPYP couldn't access the GPUs. I've fixed the problem now.
